test/script/RebalanceScUsdcV2.s.t.sol

```diff
@@ -0,0 +1,532 @@
+        assertTrue(vault.totalDebt() > 0, "total debt");
+        assertTrue(vault.totalCollateral() > 0, "total collateral");
+        assertTrue(vault.usdcBalance() > 0, "usdc balance");
+
+        script.setMorphoTargetLtv(script.morphoTargetLtv() - 0.1e18);
+        script.setAaveV2TargetLtv(script.aaveV2TargetLtv() - 0.2e18);
+
+        uint256 morphoExpectedCollateral = vault.getCollateral(morpho.id());
+        uint256 moprhoExpectedDebt =
+            priceConverter.usdcToEth(morphoExpectedCollateral).mulWadDown(script.morphoTargetLtv());
+        uint256 aaveV2ExpectedCollateral = vault.getCollateral(aaveV2.id());
+        uint256 aaveV2ExpectedDebt =
+            priceConverter.usdcToEth(aaveV2ExpectedCollateral).mulWadDown(script.aaveV2TargetLtv());
+
+        script.run();
+
+        assertApproxEqRel(vault.getCollateral(morpho.id()), morphoExpectedCollateral, 0.001e18, "morpho collateral");
+        assertApproxEqRel(vault.getCollateral(aaveV2.id()), aaveV2ExpectedCollateral, 0.001e18, "aave v2 collateral");
+        assertApproxEqRel(vault.getDebt(morpho.id()), moprhoExpectedDebt, 0.001e18, "morpho debt");
+        assertApproxEqRel(vault.getDebt(aaveV2.id()), aaveV2ExpectedDebt, 0.001e18, "aave v2 debt");
+    }
+
+    function test_run_twoAdaptersOneLeverageDownOtherUp() public {
+        script.setMorphoInvestableAmountPercent(0.5e18);
+        script.setAaveV2InvestableAmountPercent(0.5e18);
+        script.setAaveV2TargetLtv(0.5e18);
+
+        script.run(); // initial rebalance
+        script = new RebalanceScUsdcV2TestHarness(); // reset script state
+
+        assertTrue(vault.wethInvested() > 0, "weth invested");
+        assertTrue(vault.totalDebt() > 0, "total debt");
+        assertTrue(vault.totalCollateral() > 0, "total collateral");
+        assertTrue(vault.usdcBalance() > 0, "usdc balance");
+
+        script.setMorphoTargetLtv(script.morphoTargetLtv() - 0.1e18);
+        script.setAaveV2TargetLtv(script.aaveV2TargetLtv() + 0.1e18);
+
+        uint256 morphoExpectedCollateral = vault.getCollateral(morpho.id());
+        uint256 moprhoExpectedDebt =
+            priceConverter.usdcToEth(morphoExpectedCollateral).mulWadDown(script.morphoTargetLtv());
+        uint256 aaveV2ExpectedCollateral = vault.getCollateral(aaveV2.id());
+        uint256 aaveV2ExpectedDebt =
+            priceConverter.usdcToEth(aaveV2ExpectedCollateral).mulWadDown(script.aaveV2TargetLtv());
+
+        script.run();
+
+        assertApproxEqRel(vault.getCollateral(morpho.id()), morphoExpectedCollateral, 0.001e18, "morpho collateral");
+        assertApproxEqRel(vault.getCollateral(aaveV2.id()), aaveV2ExpectedCollateral, 0.001e18, "aave v2 collateral");
+        assertApproxEqRel(vault.getDebt(morpho.id()), moprhoExpectedDebt, 0.001e18, "morpho debt");
+        assertApproxEqRel(vault.getDebt(aaveV2.id()), aaveV2ExpectedDebt, 0.001e18, "aave v2 debt");
+    }
+
+    function test_run_leverageDownByAddingMoreCollateral() public {
```

Comment:
What is the difference between this test and the test below `test_run_leverageDownByAddingMoreCollateralAndRepayingDebt` ?

Reply:
this one only adds more collateral to lower the ltv (doesn't change the debt) and the second one does that + repaying some debt at the same time to lower it even further

Reply:
Is there any way to add assertions to check the repaying of debt is happening instead of just adding collateral?

Because the assertions are same in both the tests. Just the morphoTargetLtv is different.

Reply:
got it. [added](https://github.com/lindy-labs/sandclock-contracts/pull/118/commits/ed552d1026f738caf7cbe3e9ed3483d60e92d116)